hadoop-project/pom.xml

```diff
@@ -37,7 +37,7 @@
     <!--Whether to proceed to next module if any test failures exist-->
     <maven.test.failure.ignore>true</maven.test.failure.ignore>
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
-    <jetty.version>9.4.48.v20220622</jetty.version>
+    <jetty.version>9.4.49.v20220914</jetty.version>
```

Comment:
This is unrelated. Are you updating Jetty because Zookeeper has jetty dependency and you're trying to harmonize them?

If so I'd suggest to exclude the jetty from zookeeper. We update Jetty frequently and do not necessarily use the same version as Zookeeper.